### DISCUSSION TITLE
Is there possibility to configure timer? Like "expire" in OpenHAB for example

### DISCUSSION THREAD

**Discussion by darkogorgievski** [rkoshak/sensorReporter#107] [Ideas]
Hi,

I was wondering if there is possibility to configure timer to actuator? I am thinking of something similar to the "expire" functionality in OpenHAB that the actuator goes to its default (probably OFF) state after some time.
Or is there a way to configure that , just in case the connection to the MQTT server is lost?

Thanks

**Comment by rkoshak** [comment]
There is currently no support for something like that in sensorReporter. I'd have to think about how that could best be implemented. I think it's technically feasible but I'd like to avoid creating a custom implementation for each and every sensor and actuator if possible. 

It'll probably be quite some time before I can get to implementing something like this given my other commitments so if anyone wants to take a stab at it just make a note here so there isn't duplicated effort.
